packages/react-app-scaffolder/app/templates/_package.json

```diff
@@ -34,104 +30,118 @@
-    "react-router-dom": "^5.1.2",
-    "@sentry/browser": "^5.11.1",
-    "react-ga": "^2.7.0"
+      "offline-plugin": "^5.0.7",
+      "@reapit/cognito-auth": "latest",
+      "@reapit/elements": "latest",
+      "@reapit/foundations-ts-definitions": "2020-02-13",
+      "dayjs": "latest",
+      "react": "^16.11.0",
+      "react-dom": "^16.11.0",
+      "react-router": "^5.1.2",
+      "react-router-dom": "^5.1.2",
+      "@sentry/browser": "^5.11.1",
+      "react-ga": "^2.7.0"
 
     <% if (redux) { %>
-    ,"react-redux": "^7.1.3",
-    "redux": "^4.0.4",
-    "redux-saga": "^1.1.3"
-    <% } else { %>
-    ,"@testing-library/react": "^10.0.1"
-    <% } %>
+    ,
+        "react-redux": "^7.1.3",
+        "redux": "^4.0.4",
+        "redux-saga": "^1.1.3"
+    <%
+      } else { %>
+    ,
+        "@testing-library/react": "^10.0.1"
+    <%
+      } %>
 
     <% if (stylesSolution === 'styledComponents') { %>
-    ,"styled-components": "^4.3.2"
-    <% } %>
-  },
-  "devDependencies": {
-    "babel-loader": "^8.0.6",
-    "@babel/core": "^7.7.2",
-    "@babel/polyfill": "^7.7.0",
-    "@babel/preset-env": "^7.7.1",
-    "@testing-library/react-hooks": "^2.0.3",
-    "@types/enzyme": "^3.9.3",
-    "@types/jest": "^24.0.13",
-    "@types/node": "^12.0.2",
-    "@types/react": "^16.8.19",
-    "@types/react-dom": "^16.8.4",
-    "@types/react-router": "^5.1.2",
-    "@types/react-router-dom": "^5.1.2",
-    "@typescript-eslint/eslint-plugin": "^2.10.0",
-    "@typescript-eslint/parser": "^2.10.0",
-    "@sentry/webpack-plugin": "^1.10.0",
-    "cross-env": "^7.0.0",
-    "enzyme": "^3.10.0",
-    "enzyme-adapter-react-16": "^1.14.0",
-    "enzyme-to-json": "^3.3.5",
-    "eslint": "^6.7.2",
-    "eslint-config-prettier": "^6.7.0",
-    "eslint-plugin-prettier": "^3.1.1",
-    "eslint-plugin-react": "^7.17.0",
-    "eslint-plugin-react-hooks": "^2.3.0",
-    "favicons-webpack-plugin": "^2.1.0",
-    "file-loader": "^3.0.1",
-    "fork-ts-checker-notifier-webpack-plugin": "^1.0.0",
-    "fork-ts-checker-webpack-plugin": "^1.3.4",
-    "html-webpack-plugin": "^3.2.0",
-    "husky": "^2.3.0",
-    "isomorphic-fetch": "^2.2.1",
-    "jest": "^25.1.0",
-    "jest-config": "^25.1.0",
-    "jest-fetch-mock": "^2.1.2",
-    "lint-staged": "^8.1.7",
-    "loader-utils": "^1.2.3",
-    "prettier": "^1.19.1",
-    "prettier-plugin-packagejson": "^2.0.1",
-    "raw-loader": "^3.1.0",
-    "serve": "^11.3.0",
-    "source-map-loader": "^0.2.4",
-    "ts-jest": "^25.2.0",
-    "ts-loader": "^6.0.1",
-    "ts-node": "^8.3.0",
-    "ts-paths-to-webpack-alias": "^0.3.1",
-    "typescript": "3.7.2",
-    "typescript-eslint": "^0.0.1-alpha.0",
-    "webpack": "^4.41.5",
-    "webpack-cli": "^3.3.2",
-    "webpack-dev-server": "^3.4.1",
-    "sass-loader": "^7.1.0",
-    "node-sass": "^4.12.0",
-    "copy-webpack-plugin": "5.1.1",
-    "style-loader": "^1.1.3",
-    "css-loader": "^3.5.1"
+    ,
+        "styled-components": "^4.3.2"
```

Comment:
One of the ticket requirements was to switch Styled Components for Linaria since that is what we are using internally. Can you do this please? Update the questions on the CLI etc

Reply:
@willmcvay  As you can see in the description, this is only a partial part of requirements. Definitely will patch it in the next PR which remove Style Components, CSS options with centrialized webpack script

Reply:
@phmngocnghia I am cool with that as long as the requirement isn't forgotten!